### PR TITLE
(DOCS-5834) More redirects

### PR DESCRIPTION
**Pull request by hestonhoffman**
Covers some of the more complicated redirects from DOCS-5834

I couldn't get the security rules ones to work, so I'll follow up with another PR after consulting the web team.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
```
Comment (hestonhoffman) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]: Switch back to false before merge

File: content/en/security/default_rules/_index.md [hunk 2]
```diff
@@ -7,12 +7,20 @@ aliases:
...
+  - /security/default_rules/aws-rds-enabled-encryption
+  - /security/default_rules/cis-aws-1.3.0-1.14
 disable_sidebar: true
 cascade:
-    algolia:
-        rank: 11
-        category: Documentation
-        subcategory: Security Detection Rules
+- _target:
+    path: /security/default_rules/google-cloud-storage-bucket-enumerated.md
```
Comment (hestonhoffman) [inline content/en/security/default_rules/_index.md:15]: This one isn't working

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+    cache_enabled: false
```
Comment (hestonhoffman) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by brett0000FF — APPROVED** [review]
LGTM. Just curious why the alias needs `/s` in `/api/latest/downtimes/s`? Almost seems like a typo, so I wanted to confirm.

**Comment by hestonhoffman** [comment]
> LGTM. Just curious why the alias needs `/s` in `/api/latest/downtimes/s`? Almost seems like a typo, so I wanted to confirm.

Yeah, it does seem like a typo but there have been multiple hits on that url specifically. Not sure where it's coming from 🤷

**Pull request merged**